solutions/instances/README.md

```diff
@@ -1,16 +1,16 @@
-# IBM Observability Instances Deployable Architecture
+# IBM Cloud observability instances deployable architecture
 
-This architecture creates the observability instances on IBM Cloud and supports provisioning the following resources:
+This deployable architecture creates observability instances in IBM Cloud and supports provisioning the following resources:
 
-- A resource group, if one is not passed in.
-- A Log Analysis instance.
-- A Cloud Monitoring instance.
-- Creates a Cloud Object Storage (COS) instance or supports using an existing one.
-- Supports creating KMS root keys in an existing KMS instance or using existing keys if creating new buckets.
-- Creates a KMS encrypted COS bucket that is required to store archived logs or using an existing bucket.
-- Creates a KMS encrypted COS bucket for setting up Activity Tracker event routing or using an existing bucket.
-- Supports configuring the Activity Tracker event routing to the Cloud Object Storage (COS) bucket and to the Log Analysis instance .
+* A resource group, if one is not passed in.
+* An IBM Cloud Log Analysis instance.
+* An IBM Cloud Monitoring instance.
+* An IBM Cloud Object Storage instance, if one does not exist.
+* An existing key management service (KMS) for the root keys, if the keys do not exist. These keys are used when Object Storage buckets are created."
+* A KMS-encrypted bucket, if one exists. Or, creates a KMS encrypted Cloud Object Storage bucket that is required to store archived logs.
+* A KMS-encrypted Object Storage bucket, if one exists, Or, creates a KMS-encrypted Object Storage bucket for setting up Activity Tracker event routing.
```

Comment:
It does not provision a bucket if an user is using an existing one. So can we reword a lil...
example - 
`creates a KMS encrypted Cloud Object Storage bucket that is required to store archived logs. Or, uses a KMS-encrypted bucket, if one exists`